diagrams-lib.cabal

```diff
@@ -130,7 +130,8 @@ Library
                        mtl >= 2.0 && < 2.3,
                        transformers >= 0.3.0 && < 0.6.0,
                        profunctors >= 5.0 && < 6.0,
-                       exceptions >= 0.6 && < 1.0
+                       exceptions >= 0.6 && < 1.0,
+                       cereal
```

Comment:
`linear-1.20.4` depends on `cereal >=0.4.1.1 && <0.6`, so I think those version bounds should be fine here as well. Is that a good idea?